packages/core/src/events/handlers/sendIssueEscalatingHandler.ts

```diff
@@ -81,52 +73,23 @@ async function sendEmail({
     },
   )
 
-  const addresses = members.map((member) => ({
-    id: member.id,
-    address: member.email,
-    name: member.name || member.email,
-    membershipId: member.membershipId,
-  }))
-
-  const batches: SendIssueEscalatingMailOptions[] = []
-
-  for (let i = 0; i < addresses.length; i += batchSize) {
-    const batchAddresses = addresses.slice(i, i + batchSize)
-    const recipientVariables: Record<string, Record<string, unknown>> = {}
-
-    batchAddresses.forEach((user) => {
-      recipientVariables[user.address] = {
-        name: user.name,
-        id: user.id,
-        membershipId: user.membershipId,
-      }
-    })
-
-    batches.push({
-      to: batchAddresses.map((u) => u.address),
-      recipientVariables,
-      currentWorkspace: workspace,
-      issue: issueData,
-    })
-  }
-
-  await Promise.allSettled(
-    batches.map(async (batch, index) => {
-      const result = await mailer.send(batch)
-
-      if (result.error) {
-        const batchSize = Array.isArray(batch.to) ? batch.to.length : 1
-        captureException(result.error, {
-          issueId: issue.id,
-          issueTitle: issue.title,
-          workspaceId: workspace.id,
-          batchIndex: index,
-          batchSize: batchSize,
-          context: 'issue_escalation_email',
-        })
-      }
-    }),
-  )
+  await mailer.sendInBatches({
```

Comment:
DRY and now this is how can send emails in batches. Meaning same email template will be sent to all the addressList. The limitation here is that a batch email can not have personalized info by email/user. In the case of this weekly email is fine.